conanfile.py

```diff
@@ -16,27 +16,38 @@ class StormEngine(ConanFile):
 
     # dependencies used in deploy binaries
     # conan-center
-    requires = ["zlib/1.2.11", "spdlog/1.9.2", "7zip/19.00", "fast_float/3.4.0", "sdl/2.0.18", "mimalloc/2.0.3",
+    requires = ["zlib/1.2.11", "spdlog/1.9.2", "fast_float/3.4.0", "sdl/2.0.18", "mimalloc/2.0.3",
     # storm.jfrog.io
     "sentry-native/0.4.13@storm/patched", "directx/9.0@storm/prebuilt", "fmod/2.02.05@storm/prebuilt"]
     # aux dependencies (e.g. for tests)
     build_requires = "catch2/2.13.7"
 
     # optional dependencies
     def requirements(self):
+        if self.settings.os == "Windows":
+            # conan-center
+            self.requires("7zip/19.00")
+        else:
+            # conan-center
+            self.requires("libsafec/3.6.0")
+            self.requires("openssl/1.1.1o")
```

Comment:
why we need openssl? 😮

Reply:
Because of stupid conan. It will require 2 different versions of openssl for building libcurl and pulseaudio: https://github.com/q4a/storm-engine/runs/6645482117?check_suite_focus=true
```
ERROR: Conflict in libcurl/7.78.0:
    'libcurl/7.78.0' requires 'openssl/1.1.1o' while 'pulseaudio/14.2' requires 'openssl/1.1.1n'.
    To fix this conflict you need to override the package 'openssl' in your root package.
CMake Error at cmake/conan.cmake:524 (message):
  Conan install failed='1'
```
I already added `"sdl2:pulse": False,` and I still don't understand: why it's going to build `pulseaudio`